docs/redirects.json

```diff
@@ -1,4 +1,6 @@
 {
+  "": "https://developers.google.com/web/tools/chrome-devtools?utm_source=dcc&utm_medium=redirect&utm_campaign=2018Q1",
+  "/": "https://developers.google.com/web/tools/chrome-devtools?utm_source=dcc&utm_medium=redirect&utm_campaign=2018Q1",
```

Comment:
i took at look at the https://cs.chromium.org/search/?q=f:redirects.json&sq=package:chromium&type=cs and it looks like we don't need this second one. just the first.

Reply:
can you nuke this second line? i'd do it but i dont have permissions.

Reply:
Done, thanks